src/Service/Caller/Masa.Contrib.Service.Caller/Internal/CallerDependExtensions.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.Contrib.Service.Caller.Internal;
+
+/// <summary>
+/// Caller dependency orchestration
```

Comment:
What's the mean of Caller dependency orchestration

Reply:
Use to support Caller nested dependencies